src/rules/indentation.js

```diff
@@ -33,30 +33,39 @@ function testStep(step, language, configuration, mergedConfiguration) {
 }
 
 function indentation(parsedFile, unused, configuration) {
+  if (!parsedFile || Object.keys(parsedFile).length === 0) {
+    return;
+  }
   var language = languageMapping[parsedFile.language];
   var mergedConfiguration = _.merge(availableConfigs, configuration);
   errors = [];
 
   // Check Feature indentation
   test(parsedFile.location, mergedConfiguration, 'Feature');
 
-  if (parsedFile.background) {
-    // Check Background indentation
-    test(parsedFile.background.location, mergedConfiguration, 'Background');
-
-    // Check Background steps
-    parsedFile.background.steps.forEach(function(step) {
-      testStep(step, language, configuration, mergedConfiguration);
-    });
-  }
+  parsedFile.children.forEach(function(child) {
+    switch(child.type) {
+    case 'Background':
+      // Check Background indentation
+      test(child.location, mergedConfiguration, 'Background');
 
-  parsedFile.scenarioDefinitions.forEach(function(scenario) {
-    // Check Scenario indentation
-    test(scenario.location, mergedConfiguration, 'Scenario');
-    scenario.steps.forEach(function(step) {
-      // Check Step indentation
-      testStep(step, language, configuration, mergedConfiguration);
-    });
+      // Check Background steps
+      child.steps.forEach(function(step) {
+        testStep(step, language, configuration, mergedConfiguration);
+      });
+      break;
+    case 'Scenario':
+      // Check Scenario indentation
+      test(child.location, mergedConfiguration, 'Scenario');
+      child.steps.forEach(function(step) {
+        // Check Step indentation
+        testStep(step, language, configuration, mergedConfiguration);
+      });
+      break;
+    default:
+      errors.push('Unknown type ' + child.type);
```

Comment:
This should be: 
```
errors.push({message: 'Unknown gherkin node type ' + child.type,
              rule        :  rule,
              line        : child.location.line)};
```
So that the formatter doesn't break (eg this line would break: https://github.com/vsiakka/gherkin-lint/blob/master/src/formatters/stylish.js#L18)